airflow/models/dagbag.py

```diff
@@ -466,6 +467,34 @@ def _bag_dag(self, *, dag, root_dag, recursive):
                         del self.dags[subdag.dag_id]
             raise
 
+    @provide_session
+    def _check_if_dupe(self, dag, session=None):
```

Comment:
```suggestion
    def _check_if_duplicate(self, dag, session=None):
```
I would suggest it's named as above for clarity

Reply:
done